packages/server/src/utils.ts

```diff
@@ -0,0 +1,3 @@
+export function generateRoomName(): string {
+  return Math.random().toString(36).substring(4);
```

Comment:
shall we use uuid lib directly instead?

Reply:
Hm... I don't think it's needed if we are not generating a uuid. Do you want the url to look like: `/game/123e4567-e89b-12d3-a456-426614174000`?

Reply:
oh I see, so let's not use uuid maybe we can find good random word generator to create cool links like this one https://www.npmjs.com/package/fantasy-name-generator